test/contracts/staking/staking-factory-events.ts

```diff
@@ -0,0 +1,48 @@
+import {BigNumber, Event} from 'ethers'
+import {StakingPoolCreatedEvent} from '../../../typechain-types/StakingPoolFactory'
+import {expect} from 'chai'
+import {StakingPoolType} from './staking-events'
+import {isAddress} from 'ethers/lib/utils'
+
+type RewardToken = {
+    token: string
+}
+
+export type ActualStakingPoolCreatedEvent = {
+    // stakingPool: string
+    treasury: string
+    creator: string
+    rewardTokens: RewardToken[]
+    stakeToken: string
+    epochStartTimestamp: BigNumber
+    epochDuration: BigNumber
+    minimumContribution: BigNumber
+    poolType: StakingPoolType
+}
+
+/**
```

Comment:
question: There is no `eventLogs` version, are you verifying that events are both in the 1events` and `eventLos` for transaction receipts?